pkg/loader/load.go

```diff
@@ -438,8 +451,39 @@ func (s *loaderImpl) execDMLs(dmls []*DML) error {
 	return errors.Trace(err)
 }
 
+func (s *loaderImpl) createMarkTableDDL() error {
+	markTableDataBase := loopbacksync.MarkTableName[:strings.Index(loopbacksync.MarkTableName, ".")]
+	createDatabaseSQL := fmt.Sprintf("create database IF NOT EXISTS %s;", markTableDataBase)
+	createDatabase := DDL{SQL: createDatabaseSQL}
+	if s.db == nil || s.loopBackSyncInfo == nil {
+		log.Error("loaderImpl db point object is nil,maybe not create mark table and database or not need creat")
+		return nil
+	}
+	if err1 := s.execDDL(&createDatabase); err1 != nil {
+		if !pkgsql.IgnoreDDLError(err1) {
```

Comment:
don't need to check for `IgnoreDDLError` here we have using `IF NOT EXISTS`
`IgnoreDDLError` is used to ignore some error cause we re-execute it at downstream DB.